control/control.yaml

```diff
@@ -107,3 +109,23 @@ components:
           type: string
           enum: [start, stop]
 
+    Control.Bgpv4ControlState:
+      description: >-
+        Control BGP Protocol state to handle special case.
```

Comment:
what does this description mean?   what is a the special case being handled?  how would a user know what that means?

Reply:
dut_ipv4_address is a property where as ipv4 is protocol stack therefore one specify as Device.Pattern where as another as Device.Ipv4.
I have remove start, stop which already handle within FlowTransmitState. Sonic has test case which require withdraw_routes and advertise_routes. Therefore add BGP route action which fulfill those requirements.